src/store/metadata/oxigraph/store.py

```diff
@@ -41,6 +41,13 @@
 class OxigraphMetadataStore(BaseMetadataStore):
     def setup(self):
         oxigraph_url = os.environ.get("GRAPH_DB_URL", None)
+        if oxigraph_url is None:
```

Comment:
can see what you're trying to do, but we cant do this, that'd be test logic inside application code we need to keep these things separated.